README.md

```diff
@@ -255,6 +255,32 @@ final interceptor = nock.get("/video")
   ..reply(200, <int>[73, 32, 97, 109, 32, 118, 105, 100, 101, 111]);
 ```
 
+### Specifying `head` method responses
+
+Under normal circumstances, a `get` interceptor will imply a corresponding `head` response:
+
+For example:
+```dart
+final interceptor = nock.get("/ping")
+  ..reply(200, "pong");
+```
+responds to both `get` and `head`.  
+
+You can suppress the standard head response as follows:
+```dart
+final interceptor = nock.get("/ping")
+  ..reply(200, "pong", autoHead: false);
```

Comment:
I know: not the `withHead()` or `noHead()` methods which we were discussing, but it seemed to capture both positive and negative direction in one.  Does this seem to you like a reasonable approach?